lib/mandrill-transport.js

```diff
@@ -23,6 +23,27 @@ MandrillTransport.prototype.send = function(mail, callback) {
   var bccAddrs = addressparser(data.bcc) || [];
   var fromAddr = addressparser(data.from)[0] || {};
   var mandrillOptions = data.mandrillOptions || {};
+  var attachments = [];
+
+  // convert attachments passed to nodemailer to the structure used by mandrill
+  if (mail.message && mail.message.childNodes) {
+    mail.message.childNodes.forEach(function(node) {
+      var headersAsMap = {};
+
+      node._headers && node._headers.forEach(function(headerEntry) {
```

Comment:
tried it and it failed for me, im running node 0.10.x, on which Array.prototype.find is not available.
i use nvm, so tried changing to nodejs 4.x and worked there.

but for sake of compatibility i left code as is, your'e welcome to refactor if you want to.